docs/gno-infra/setting-up-a-faucet.md

```diff
@@ -0,0 +1,173 @@
+---
+id: setting-up-a-faucet
+---
+
+# Setting up a faucet for your Gno network
+
+In this tutorial, we will cover how to run a local native currency faucet that 
+works seamlessly with a Gno node. Using the faucet, any address can get a hold
+of testnet GNOTs.
+
+## Prerequisites
+- Git
+- Go 1.21+
+- Make (for running Makefiles)
+- `gnoland` & `gnokey` installed
+- A Gno.land keypair generated using [`gnokey`](../gno-tooling/cli/gnokey.md)
+
+## Premining funds to an address
+
+Before setting up the faucet, we need to make sure that the address will be used
```

Comment:
This is not a correct English sentence:

`we need to make sure that the address will be used to serve the funds contain enough testnet funds`

Reply:
This section was completely removed as per https://github.com/gnolang/gno/pull/2122#discussion_r1625489784